using-valtimo/plugin/portaaltaak/configure-portaaltaak-plugin.md

```diff
@@ -0,0 +1,118 @@
+# Using the Portaal taak Plugin
+
+The Portaal Taak plugin is used to generate "taak" (task) objects within the Objecten API for a BPMN 
+user task. As a result, such tasks can be completed via external applications, such as the NL Portal. This feature 
+empowers Valtimo to assign tasks to other systems that are more adept at performing them. Furthermore, it enables users 
+who lack access to the Valtimo frontend to access these tasks.
+
+## How does the plugin work
+
+The lifecycle of a portaal taak is as follows:
+1. When a process instance reaches a user task configured with the [Create Portaal taak](#create-portaal-taak) action
+an instance of the task is created in Valtimo as usual. Additionally a taak object is created in the Objecten API of
+object type `Taak`. The status of the taak object will be `open`. The taak object includes information on who the 
+intended recipient is. This is the person or organization that is supposed to complete the task.
+2. External applications, such as the NL Portal, can retrieve the task information and offer a way to 
+complete the task. Often this will be a form to input data. 
+3. When the user completes the task in the external system, this system will update the taak object with the new data
+and change the status in the object to `ingediend`.
+4. The Objecten API should send a notification to the Notificaties API indicating the the object has been updated.
+5. Valtimo should be subscribed to notifications from the Notificaties API as part of the plugin setup and will receive
+the notification. When an update notification for a taak object is received Valtimo will check the status. If this is 
+set to `ingediend` the updated data from the taak object will be stored based on the value resolver configuration, and 
```

Comment:
Maybe link to the relevant section on how this can be configured?

Reply:
I tried to find this but i don't think we have any documentation on this.